.github/workflows/send-email.yml

```diff
@@ -0,0 +1,95 @@
+name: Send communications
+on:
+  push:
+    branches:
+      - production
+    paths:
+      - 'apps/dialtone-documentation/docs/about/whats-new/posts/*.md'
+
+env:
+  HUSKY: 0
+
+jobs:
+  changed-files:
+    name: Get changed files
+    runs-on: ubuntu-latest
+    outputs:
+      added-posts: ${{ steps.changed-blog-posts.outputs.added_files }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Get changed blog posts
+        id: changed-blog-posts
+        uses: tj-actions/changed-files@v42
+        with:
+          files: 'apps/dialtone-documentation/docs/about/whats-new/posts/*.md'
+          json: true
+          escape_json: false
+      - name: List all changed files
+        run: echo '${{ steps.changed-blog-posts.outputs.all_changed_files }}'
+
+  send-email:
+    name: Send email
+    runs-on: ubuntu-latest
+    needs: [changed-files]
+    strategy:
+      matrix:
+        files: ${{ fromJSON(needs.changed-files.outputs.added-posts) }}
+    env:
+      FROM_MAIL: ${{ secrets.MAIL_ACCOUNT }}
+      TO_MAIL: dialtone-users@dialpad.com
+      FILE_PATH: ${{ matrix.files }}
+      REPLY_TO: dialtone@dialpad.com
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Get link from file name
+        id: get-post-url
+        run: |
+          PREFIX="https:\/\/dialtone.dialpad.com"
+          echo "POST_URL=$(echo $FILE_PATH | sed -e "s/apps\/dialtone-documentation\/docs/$PREFIX/" -e 's/md/html/')" >> $GITHUB_ENV
+
+      - name: Get heading from blog post
+        id: get-heading
+        run: |
+          echo "BLOG_POST_HEADING=$(sed -n 's/^heading: *//p' $FILE_PATH)" >> $GITHUB_ENV
+
+      # This will do the following:
+      # - Remove frontmatter.
+      # - Remove script setup.
+      # - Remove blog post component.
+      # - Remove comments.
+      # - Remove empty lines at the beginning of the file.
+      # - Ellipsis the article to the first 10 lines.
+      # - Append the URL of the article at the end of the article.
+      - name: Process Markdown file
+        run: >
+          sed -i
+          -e '/^---$/,/^---$/d'
+          -e '/^<script setup>$/,/^<\/script>$/d'
+          -e '/^<\/*BlogPost.*$/d'
+          -e '/^<!--.*$/d'
+          -e '/./,$!d'
+          $FILE_PATH &&
+          sed -i 10q $FILE_PATH &&
+          echo -e "... Read the full article: $POST_URL" >> $FILE_PATH;
+
+      - name: Send email
+        uses: dawidd6/action-send-mail@v3
+        with:
+          connection_url: ${{ secrets.MAIL_CONNECTION }}
+          subject: "Dialtone: New blog post '${{ env.BLOG_POST_HEADING }}'"
```

Comment:
Let's change the subject to `[dialtone] New blog post: ${{ env.BLOG_POST_HEADING }}` Since this seems to be the standard way of categorizing emails at dialpad. Makes it easier to add to filters for it.